docs/changelog.mdx

```diff
@@ -4,6 +4,27 @@ sidebar_label: 'Changelog'
 
 # Changelog
 
+## Release 0.0.29
+
+_October 9th, 2024_
+
+**New Features**
+- Additional game and environment metadata is now captured during gameplay recording (e.g. Unity version, active input devices, etc.)
+- Bot Sequences can now be paused
```

Comment:
This is also possible when running Bot Segments individually (using the hotkey combo)
I was unsure of this, but I just gave it a test and it works

Reply:
Added!